src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/records/SimpleCount.java

```diff
@@ -22,6 +24,20 @@ public SimpleCount(final SimpleInterval interval,
         this.count = ParamUtils.isPositiveOrZero(count, "Can't construct SimpleCount with negative count.");
     }
 
+    public SimpleCount(final NamedFeature namedFeature) {
+        this.interval = new SimpleInterval(namedFeature.getContig(), namedFeature.getStart(), namedFeature.getEnd());
+        try {
+            int count = Integer.parseInt(namedFeature.getName());
+            if (count > 0) {
+                this.count = count;
+            } else {
```

Comment:
I'm not sure these user exceptions for your specific SimpleCount case belong down here in this constructor, especially since you are using a non-standard NamedFeature usage. Pull this logic out into a method and call the other constructor directly so the exceptions stay associated with the HaplotypeCaller and don't trip anybody up in the future.

Reply:
As per our conversation, I trimmed down the code here a little and switched to using `IllegalArgumentException` so as to not assume the user is the source of a parsing error. It now nicely handles both the unable to parse integer and not-positive error cases nicely.